src/main/java/net/neoforged/neoforge/common/data/LanguageProvider.java

```diff
@@ -114,11 +115,15 @@ public void addTag(Supplier<? extends TagKey<?>> key, String name) {
     }
 
     public void add(TagKey<?> tagKey, String name) {
-        add(Tags.getTagTranslationKey(tagKey), name);
+        add(net.neoforged.neoforge.common.Tags.getTagTranslationKey(tagKey), name);
```

Comment:
Why is this being changed from an import to an FQN?

Reply:
I might have messed up the original rebase but should be fixed now